appserver/payara-appserver-modules/microprofile/telemetry/src/main/java/fish/payara/microprofile/telemetry/tracing/WithSpanMethodInterceptor.java

```diff
@@ -219,4 +217,38 @@ private void extractSpanAttributes(final InvocationContext invocationContext, fi
                         InvocationContext::getParameters);
         extractor.onStart(builder, Context.current(), invocationContext);
     }
+
+    private Object handleSyncInvocation(InvocationContext invocationContext, Span span) throws Exception {
+        try (var ignore = PropagationHelper.start(span, Context.current())) {
+            try {
+                return invocationContext.proceed();
+            } catch (final Exception ex) {
+                printExceptionAttributes(span, ex);
```

Comment:
`print` is bit surprising verb here for me. I'd go for something starting with `set` or `apply` or `markSpanAsFailed`